src/styles/blocks/header.scss

```diff
@@ -0,0 +1,80 @@
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  display: flex;
+  margin: auto 50px;
+}
+
+.header__menu {
+  display: flex;
+  margin: 0 50px;
+}
+
+.header__menu-list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.header__menu-item {
+  margin: 0 10px;
+}
+
+.header__menu-list li:first-of-type {
+  margin-left: 0;
+}
+
+.header__menu-list li:last-of-type {
+  margin-right: 0;
+}
+
+.header__menu-link {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #060b35;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  position: relative;
+  display: block;
+}
+
+.header__menu-link:hover {
+  color: #00acdc;
+}
+
+.header__menu-link:active {
+  color: #00acdc;
+}
+
+.header__menu-link:active::after {
+  position: absolute;
+  display: block;
+  width: 100%;
+  height: 4px;
+  content: "";
+  background: #00acdc;
+  border-radius: 8px;
+}
```

Comment:
please use nesting here
checklist 10 [SASS]https://github.com/mate-academy/layout_catalog/blob/master/checklist.md

Reply:
@valerii-derkach Hello, thank you! I have fixed the bugs.